content/en/blog/_posts/2024-11-08-kubernetes-1.32-sneak-peek.md

```diff
@@ -0,0 +1,143 @@
+By extending the DRA framework, this enhancement allows workloads to request and consume portions of these resources, increasing utilization and scheduling flexibility. The proposed enhancement also provides primitives for representing both full devices and their partitions more compactly, supporting the recent transition from "classic DRA" to the "structured parameters" approach. 
+The newly proposed extensions, fully transparent to the end-user, enable vendors to provision “overlapping” partitions, ensuring that the scheduler does not allocate conflicting partitions. This approach allows vendors to dynamically create new partitions after allocation, eliminating the need for pre-partitioned configurations while still supporting device partition selection through existing mechanisms like `ResourceClaim` and `ResourceSlice` in Kubernetes v1.31.
+
+**Resource Claim status and standardized network interface data**
+
+KEP [#4817](https://github.com/kubernetes/enhancements/issues/4817) adds driver-owned fields in `ResourceClaim.Status` with the possibility of having standardized network interface data.
+This proposal enhances the `ResourceClaim.Status` field by adding a new sub-field: `Devices`.
+The new entry allows the reporting of driver-specific device status data for each allocated device in a `ResourceClaim`. Allowing the drivers to report such data will improve observability and troubleshooting and enable new functionalities such as network services (in case the IP addresses of a network device are reported).
+
+
+**DRA Structured Parameters** 
+
+The original dynamic resource allocation (DRA) uses claim and class parameters that are opaque to Kubernetes. KEP [#4381](https://github.com/kubernetes/enhancements/issues/4381) introduces structured parameters so that kube-scheduler and Cluster Autoscaler can handle a claim allocation without relying on a third-party driver.
+As users increasingly adopt Kubernetes as their management solution for batch-processing workloads and edge computing, the need to expose specialized hardware to Pods is increasing.
+Such workloads no longer need just RAM and CPU, so this KEP introduced a new API to describe which of these specialized resources a Pod needs, such as devices like a GPU or other kinds of accelerators.
+
+
+**Prioritized alternatives in device requests** 
+
+KEP [#4816](https://github.com/kubernetes/enhancements/issues/4816) adds support for a prioritized list of selection criteria attached to a device request in a `ResourceClaim`.
+The DRA's Structured Parameters feature has added the ability to make requests for very specific types of devices using a `ResourceClaim`.
```

Comment:
How about:
```suggestion
KEP [#4816](https://github.com/kubernetes/enhancements/issues/4816) proposes support for prioritized
selection criteria associated to a device request in a ResourceClaim.
DRA _structured parameters_ will gain the ability to make requests for very specific types of devices using a ResourceClaim.
```

Reply:
Perfect, now I also see what you meant with API Objects withput the backtick. 

A teaching for next blog :)